product_docs/docs/pem/8/pem_query_tool.mdx

```diff
@@ -238,26 +238,26 @@ Use the `Connection status` feature to view the current connection and transacti
 
 ## Macros
 
-Query Tool Macros enable you to execute pre-defined SQL queries with a single key press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
+Query Tool Macros enable you to execute pre-defined SQL queries with a single key-press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
 
 ![Query Tool - Connection Status button](images/query_tool_manage_macros.png)
 
-To create a macro, select the *Manage Macros* option from the *Macros* menu on the *Query Tool*. Select the key you wish to use, enter the name of the macro, and the query, optionally including the selection placeholder, and then click the *Save* button to store the macro.
+To create a macro, select the `Manage Macros` option from the Macros menu on the Query Tool. Select the key you wish to use, enter the name of the macro, and the query, optionally including the selection placeholder, and then click the `Save` button to store the macro.
```

Comment:
This was a complicated way to refer to a menu option. Don't use wish, use want. This is also a multi-step list, so I would do it like this:

To create a macro:

1.  In the Query Tool, select **Macros > Manage Macros**. 
1.  Select the key you want to use, and enter the name of the macro and the query. 
1.  Optionally, include the selection placeholder. 
1.  Select **Save**.